subsys/bluetooth/services/dis.c

```diff
@@ -101,6 +138,74 @@ static ssize_t read_pnp_id(struct bt_conn *conn,
 }
 #endif
 
+#if CONFIG_BT_DIS_SYSTEM_ID
+static ssize_t read_system_id(struct bt_conn *conn,
+			   const struct bt_gatt_attr *attr, void *buf,
+			   uint16_t len, uint16_t offset)
+{
```

Comment:
@pdgendt the clang-format suggestion here looks a bit weird due to tabs, as the suggested result is not properly aligned. Is there anything we can do to improve that, or is this just verbose from the tool?

Reply:
Hmn, the alignment is off-by-one because of the `+` sign.. Not sure how we could improve here.

Reply:
That was also my thought. That's what we get from mixing tabs and spaces 8)